src/briefcase/config.py

```diff
@@ -216,7 +218,7 @@ def merge_config(config, data):
         is modified in-situ.
     :param data: The new configuration data to merge into the configuration.
     """
-    for option in ['requires', 'sources']:
+    for option in ['requires', 'sources', 'is_supported']:
```

Comment:
Yes, it should be `supported`, I'll fix that!

Reply:
Actually, I will remove the `supported` element from that list as this would result in the `supported` field being `true` even with the `supported=false` attribute in the platform specific toml attributes. Removing it altogether results in the intended outcome.